src/load

```diff
@@ -398,25 +404,24 @@ def save_epss_data(appctx, args, db_records):
         bar_format = '{n_fmt}/{total} {l_bar}{bar}| ({elapsed}/{remaining})'
         db_insert_progress = tqdm(total=len(db_records), bar_format=f'{db_insert_progress_name:<20} {bar_format}',ascii=True)
 
-        for batch in [list(db_records.keys())[i:i + BATCH_SIZE] for i in range(0, len(db_records), BATCH_SIZE)]:
+        BATCH_SIZE_epss = 1024
+        for batch in [list(db_records.keys())[i:i + BATCH_SIZE_epss] for i in range(0, len(db_records), BATCH_SIZE_epss)]:
 
-            vuln_query= session.query(Vuln).filter(Vuln.vuln_id.in_(batch)).options(load_only(Vuln.data))
+            vuln_query= session.query(Vuln).filter(Vuln.vuln_id.in_(batch)).options(load_only(Vuln.data)).all()
             update_values = []
 
-            for vuln_record in vuln_query.all():
-
+            for vuln_record in vuln_query:
                 epss_desired_row = db_records[vuln_record.vuln_id]       # Find the corresponding record in the batch
 
                 if 'epss' in vuln_record.data['metrics']:           # Check if 'epss' key already exists in data.metrics
```

Comment:
there is no point to check if the EPSS if exists and to do update or assignment, in  both cases an assignment is the accepted approach as there are no other attributes that needs to be kept with previous values. 
Please add spaces after commas for better code reading.